packages/web-components/src/utils/helpers.ts

```diff
@@ -229,7 +229,9 @@ export const isEmptyString = (value: string): boolean => {
  */
 export const getLabelFromValue = (
   value: string,
-  options: IcMenuOption[]
+  options: IcMenuOption[],
+  valueField: string,
```

Comment:
Could you make these parameters optional, so that components like ic-select don't have to provide dummy values if they don't utilise custom fields?